app/scripts/components/common/page-header/styles.scss

```diff
@@ -0,0 +1,25 @@
+@use '$styles/veda-ui-theme-vars.scss' as themeVars;
+
+.usa-logo {
+  max-width: none !important;
+}
+
+.usa-nav__secondary-item {
+  font-weight: themeVars.$veda-uswds-fontweight-regular;
+}
+
+@media (width >= 64em) {
+  .usa-nav__secondary-links {
+    margin-bottom: 1.5rem;
+  }
+
+  .usa-nav__secondary-item .usa-nav__link  {
+    padding: 0;
```

Comment:
I thought uswds has enough utility classes that we don't need these individual styles for each item - https://designsystem.digital.gov/utilities/margin-and-padding/ any reason not using these utility classes?

Reply:
yeah, if you are talking about doing something like ...
```
        <USWDSExtendedNav
          primaryItems={primaryItems}
          secondaryItems={secondaryItems}
          mobileExpanded={expanded}
          onToggleMobileNav={toggleExpansion}
          className='padding-0'
        />
```
unfortunately it doesn't work because this component has nested elements and this wants to just target the secondary nav items, not the primary. If you know how to or mean something else though, please suggest. Otherwise this follows this [pattern](https://github.com/NASA-IMPACT/veda-ui/blob/main/app/scripts/components/common/datepicker/index.scss#L53-L55) more closely.